File: competidores/P11/maior.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <string.h>

int main(){
   int n1,n2,n3,n4;
   double resp1=0,resp2=0;
   char op1,op2;

        scanf("%d %c %d %d %c %d",&n1, &op1, &n2, &n3, &op2, &n4);

        if(op1== 'x'){
            resp1=n1*n2;
        }
        if(op1== '+'){
            resp1=n1+n2;
        }
        if(op1== '^'){
            resp1=pow(n1,n2);
        }

        if(op2== 'x'){
            resp2=n3*n4;
        }
        if(op2== '+'){
            resp2=n3+n4;
        }
        if(op2== '^'){
            resp2=pow(n3,n4);
        }


        if(resp1<resp2){
            printf("<");
        }else if(resp1>resp2){
            printf(">");
        }else{
            printf("=");
        }
return 0;
}

File: competidores/P04/maior.java
import java.util.Scanner;

public class maior {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a = sc.nextInt();
		String op1 = sc.next();
		int b = sc.nextInt();
		int c = sc.nextInt();
		String op2 = sc.next();
		int d = sc.nextInt();
						
		int num1 = 0;
		int num2 = 0;
		
		if(op1.equals("+")){
			num1 = a+b;
		}else if(op1.equals("x")){
			num1 = a*b;
		}else if(op1.equals("^")){
			num1 = (int)Math.pow(a, b);
		}
				
		if(op2.equals("+")){
			num2 = c+d;
		}else if(op2.equals("x")){
			num2 = c*d;
		}else if(op2.equals("^")){
			num2 = (int)Math.pow(c, d);
		}
		
		if(num1>num2){
			System.out.println(">");
		}else if(num1<num2){
			System.out.println("<");
		}else{
			System.out.println("=");
		}
		
		sc.close();
	}

}
